lib/src/util/stack_trace_mapper.dart

```diff
@@ -17,14 +18,69 @@ class StackTraceMapper {
   /// The URI of the SDK root from which dart2js loaded its sources.
   final Uri _sdkRoot;
 
-  StackTraceMapper(String contents,
+  /// The contents of the source map.
+  final String _contents;
+
+  /// The URI of the source map.
```

Comment:
Nit: "URL" (if you can fix the `_sdkRoot` docs too that would be great)

Technically, [URI is a deprecated term](https://url.spec.whatwg.org/#goals)!